packages/components/check-tag/src/check-tag.vue

```diff
@@ -1,42 +1,30 @@
 <template>
-  <span
-    :class="{
-      'el-check-tag': true,
-      'is-checked': checked,
-    }"
-    @click="onChange"
-  >
+  <span :class="[ns.b(), { 'is-checked': checked }]" @click="onChange">
```

Comment:
Can we move this into `setup` so that we can have a cleaner template?

Reply:
I think we should put class and style to **template**. `script` is best to put logic related.

Reply:
Yes I agree, but for here **Computing the classnames is sort of part of the logic here**, I think a cleaner template would have better readability, even though in this case the template is relatively simple, but we kind of need this to follow the code style (making template cleaner)